deploy/crds/aws_v1alpha1_awsfederatedrole_readonly_cr.yaml

```diff
@@ -4,8 +4,8 @@ metadata:
   name: read-only
   namespace: aws-account-operator
 spec:
-  roleDisplayName: read-only
-  roleDescription: This is the description of read-only
+  roleDisplayName: Read Only
+  roleDescription: User may view all AWS resources for this cluster, but not modify or create resources.
```

Comment:
"User may view all AWS resources for this cluster, but not modify, create or delete resources".